Gems/Multiplayer/Code/Source/Editor/MultiplayerEditorSystemComponent.cpp

```diff
@@ -610,4 +625,16 @@ namespace Multiplayer
             AZ::ComponentTypeList{ azrtti_typeid<NetBindComponent>(), azrtti_typeid<NetworkTransformComponent>() }
         );
     }
+    void MultiplayerEditorSystemComponent::OnStopPlayInEditorBegin()
+    {
+        if (!editor_clientserver)
```

Comment:
You might want to check AgentType here vs the CVAR in the unlikely but possible case of someone editing editor_clientserver during play mode.

Reply:
Good idea, added a check.